catalog/ui/search-ui/search-endpoint/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -23,6 +23,9 @@
         <AD description="Disables relevance and distance normalization." name="Disable Normalization"
             id="normalizationDisabled" required="false" type="Boolean" default="false"/>
 
+        <AD description="Controls the maximum number of results returned from a query." name="Maximum Query Results"
```

Comment:
Tried to explain a bit more. Please review the new text and let me know what else should be altered.